app/frontend/src/features/profile/edit/EditHostingPreference.test.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  render,
+  screen,
+  waitFor,
+  queryByAttribute,
+  fireEvent,
+  act,
+} from "@testing-library/react";
+import { Route, Switch } from "react-router-dom";
+import { addDefaultUser, MockedService } from "../../../test/utils";
+import wrapper from "test/hookWrapper";
+import EditHostingPreference from "./EditHostingPreference";
+import { getUser } from "test/serviceMockDefaults";
+import { service } from "service";
+import { profileRoute } from "routes";
```

Comment:
Can you run `yarn lint --fix` so the imports are sorted?

Reply:
I ran `yarn format`. I thought that will do the lint issues and all. Anyway I did this.